test/format/format.h

```diff
@@ -102,6 +102,12 @@ typedef struct {
     const char *track; /* Tag for tracking operation progress */
 } SAP;
 
+/* Arguments to the read scanner. */
+typedef struct {
+    WT_CONNECTION *conn;
+    WT_RAND_STATE *rnd;
+} READ_SCAN_ARGS;
```

Comment:
Alphabetize above RWLOCK in list of random format structures.

Reply:
This is done.